modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/ReloadingDatabasesWhilePerformingGeoLookupsIT.java

```diff
@@ -146,13 +158,14 @@ public void test() throws Exception {
         for (DatabaseReaderLazyLoader lazyLoader : databaseRegistry.getAllDatabases()) {
             assertThat(lazyLoader.current(), equalTo(0));
         }
+        // Avoid accumulating many temp dirs while running with -Dtests.iters=X
+        IOUtils.rm(geoIpModulesDir, geoIpConfigDir, geoIpTmpDir);
```

Comment:
+100 for that, I clogged up my disk running this test with `tests.iters`

Reply:
same here :)